cmd/konnect_dump.go

```diff
@@ -31,6 +31,14 @@ configure Konnect.` + konnectAlphaState,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		httpClient := http.DefaultClient
 
+		ok, err := confirmFileOverwrite(dumpCmdKongStateFile, dumpCmdStateFormat, assumeYes)
+		if err != nil {
+			return err
+		}
+		if !ok {
+			return nil
+		}
+
```

Comment:
Could this block be made identical to the equivalent one in dump.go?

Reply:
Updated.